opencascade/fix-private-linking.patch

```diff
@@ -1,11 +1,11 @@
-Index: opencascade-7.4.0/adm/cmake/occt_toolkit.cmake
+Index: opencascade-7.6.2/adm/cmake/occt_toolkit.cmake
 ===================================================================
---- opencascade-7.4.0.orig/adm/cmake/occt_toolkit.cmake
-+++ opencascade-7.4.0/adm/cmake/occt_toolkit.cmake
-@@ -354,7 +354,7 @@ else()
- endif()
- 
- if (BUILD_SHARED_LIBS)
+--- opencascade-7.6.2.orig/adm/cmake/occt_toolkit.cmake
++++ opencascade-7.6.2/adm/cmake/occt_toolkit.cmake
+@@ -400,7 +400,7 @@
+   if(IS_VTK_9XX)
+     string (REGEX REPLACE "vtk" "VTK::" USED_TOOLKITS_BY_CURRENT_PROJECT "${USED_TOOLKITS_BY_CURRENT_PROJECT}")
```

Comment:
Hm, I don't understand what this conditional and this line is for??

Reply:
I didn't write this patch manually, but created it using `git diff`. The patch file contains the lines *preceeding* the actually modified lines, which are given in line 9 and 10 below. The patch makes sure that transient dependencies are linked privately.

So this conditional wasn't written by me, but is part of opencascade.

Reply:
I see, thank you!